src/adapters/price-updater.ts

```diff
@@ -46,54 +47,44 @@ const getFiatExchangeRatesResponseParser = StrictSchema<ApiExchangeRateResponse>
   })
 );
 
-const parsedReactAppPriceUpdaterApiUrl = z
-  .string()
-  .safeParse(process.env.REACT_APP_PRICE_UPDATER_API_URL);
-
-const parsedReactAppPriceUpdaterApiKey = z
-  .string()
-  .safeParse(process.env.REACT_APP_PRICE_UPDATER_API_KEY);
-
 /**
  * Returns a list of tokens with usd price.
  */
 function getTokensPrice(): Promise<Token[]> {
-  if (parsedReactAppPriceUpdaterApiUrl.success === false) {
-    return Promise.reject(parsedReactAppPriceUpdaterApiUrl.error.message);
-  } else if (parsedReactAppPriceUpdaterApiKey.success === false) {
-    return Promise.reject(parsedReactAppPriceUpdaterApiKey.error.message);
-  } else {
+  const eitherEnv = adapters.env.getEnv();
```

Comment:
Doing this we are going to parse the `process.env` each time that we need an env var. I would parse it at the beginning of the application and store the result in the `global` store.

Reply:
Right, but parsing the `process.env` is so cheap that I don't see the point of storing it already parsed. Store it implies extra code and complexity that we don't need for such a cheap parser.

Reply:
Well, the extra code is just 3 redux actions and 1 thunk... I prefer to do this just one when the app loads instead of having to parse it each time that i need to access the env

Reply:
Ok. I will implement the 3 redux actions and the thunk on global and create a `useEffect` in `app.view`.

Reply:
Done